ui/lib/util.js

```diff
@@ -111,3 +111,18 @@ exports.getPubStats = function (peers) {
     hasSyncIssue: (!membersof || (!membersofUntried && !membersofActive))
   }
 }
+
+exports.getRelayPeerIds = function (peers) {
```

Comment:
Is this a function to get pubs, or just peers you connect to? If the latter, this ought to be more like... `getConnectedPeerIds`